docs/scripts/d2_svg_file.py

```diff
@@ -9,6 +9,8 @@
 from mkdocs.structure.pages import Page
 from mkdocs.utils import write_file
 
+IGNORE_HOOKING_FOR = []
```

Comment:
I originally was going to disable hooking for my new diagram, but eventually decided the other way. I think this code is useful to keep in case we want to disable this in the future. For example, if we have diagrams that embed clickable links.